settings.js

```diff
@@ -211,6 +211,7 @@ window.spinnakerSettings = {
     required: false,
   },
   pubsubProviders: ['google'], // TODO(joonlim): Add amazon once it is confirmed that amazon pub/sub works.
+  plugins: [],
```

Comment:
can you update this to:
```suggestion
  plugins: {%plugins%},
```